### ISSUE TITLE
Spam scores not showing

### ISSUE THREAD

**Issue by palashahuja** [moisseev/rspamd-spamness#2]
@moisseev  This is the output that I am getting as of now for the addon

![screenshot from 2016-01-01 16 52 01](https://cloud.githubusercontent.com/assets/8504351/12070654/61b7b76c-b0a8-11e5-86ac-ce2c48414ced.png)

**Comment by moisseev** [comment]
Do messages in the folder have the `X-Spamd-Result` header? Did you re-index the `Spam` folder after the add-on installation?

**Comment by hufon** [comment]
I get the same issue here's a message source 
`Return-Path: <b083049c976hubert=fongarnand.fr@bounce.twitter.com>
Delivered-To: <hubert@fongarnand.fr>
Received: from mail.fongarnand.fr ([127.0.0.1])
	by piyubserver (Dovecot) with LMTP id LLPiIj3GrFbsHQAA4JhRsg
	for <hubert@fongarnand.fr>; Sat, 30 Jan 2016 15:18:37 +0100
Received: from spruce-goose-af.twitter.com (spruce-goose-af.twitter.com [199.59.150.75])
	by mail.fongarnand.fr (Postfix) with ESMTPS id B4C4011A00A2
	for <hubert@fongarnand.fr>; Sat, 30 Jan 2016 15:18:35 +0100 (CET)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=twitter.com;
	s=dkim-201406; t=1454163512;
	bh=W8xYI4BrDH3TLDl4fCesA4UBAuaPAIrXcU+zxVrkVJk=;
	h=Date:From:To:Subject:MIME-Version:Content-Type:List-Unsubscribe:
	 Message-ID;
	b=glA0lwnPB4zoVU59vRHrg5a9zrp1KBFBLw6X2K5TzE0c2/VG4EazVP8Ij7+mSv20k
	 XtpnrqDyBQf0JkJEtqN9k29WT3VOOEVD9UEVSjjLTpDjKPWL9cYXflBS84+mNDAVJE
	 3OlK4NGt0XjGwfuZlSKidgPJmBngcbCxYr/XQRIJI3DPOKRBobgXy1OliWo0Gnn35v
	 28fO5PAgNQ+igxqbdxIT0v5Nbgc+mW5Qwcq4ecLQ4OeE6b17VRuQ9Z3lBn20IZYSzF
	 wZt4HPqRr46ByHzvrLqeiV8ru/hobcdORmgXWcxlMgO783X0Cl9W1BP9KX7VOCh303
	 HM6BozjzgJeMg==
X-MSFBL: eyJiIjoic21mMS1iZXAtMzgtc3IxLUV2ZXJ5dGhpbmcuMTc1IiwiciI6Imh1YmVy
	dEBmb25nYXJuYW5kLmZyIiwiZyI6IkV2ZXJ5dGhpbmciLCJ1IjoiaHViZXJ0QGZv
	bmdhcm5hbmQuZnJAaWlkIyM2MDI5OTk4MThjMGQ0NWE1OTc4N2UzNmU5N2JjYzE3
	OEB1c2IjIzI0QDI0NEAyMzA2NzM5M0AwQGMwMWRhYmRkZTgyN2E3MDQzYzVkMWZk
	ZTNjYjI0ZTJiMjI3ZjkwZTQifQ==
Date: Sat, 30 Jan 2016 14:18:32 +0000
From: =?UTF-8?Q?Populaire_au_sein_de_votre_r=C3=A9seau?= <info@twitter.com>
To: Hubert FONGARNAND <hubert@fongarnand.fr>
Subject: =?UTF-8?Q?Papa&Maman_a_tweet=C3=A9_:_Un_renard_sur_mon_mur_!__#renard_#?=
 =?UTF-8?Q?foxaddict_#fox_#decomurale_#papaetmaman_#boutiqueenligne?=
MIME-Version: 1.0
Content-Type: multipart/alternative; 
	boundary="----=_Part_18031827_1805345715.1454163512768"
List-Unsubscribe: https://twitter.com/i/u?t=1&listunsub=t&cn=ZmxleGlibGVfcmVjcw%3D%3D&sig=98d2824bbb5a29be660d061b57441e7fedba7e21&iid=602999818c0d45a59787e36e97bcc178&uid=23067393&nid=244+26&usbid=6
Feedback-ID: 0040162518f58f41d1f0:15491f3b2ee48656f8e7fb2fac:none:twitterESP
Precedence: Bulk
Message-ID: <CF.98.57932.836CCA65@twitter.com>
X-Spamd-Result: default: False [-9.57 / 15.00]
 WHITELIST_SPF_DKIM(-3.00)
 R_SPF_ALLOW(-1.50)
 HFILTER_URL_ONLY(1.50)
 MIME_GOOD(-0.10)
 ONCE_RECEIVED(0.10)
 DMARC_POLICY_ALLOW(-0.50)
 R_PARTS_DIFFER(0.61)
 URIBL_BLOCKED(0.00)
 RWL_MAILSPIKE_POSSIBLE(0.00)
 WHITELIST_DMARC(-7.00)
 BAYES_SPAM(1.11)
 R_DKIM_ALLOW(-1.10)
 FORGED_SENDER(0.30)
X-Spamd-Server: localhost
X-Spamd-Scan-Time: 0.85
X-Spamd-Queue-ID: B4C4011A00A2`

**Comment by moisseev** [comment]
Nothing wrong with the header of your message source, assuming it just a formatting issue and in the actual message lines with symbols start with spaces like that:
~~~
X-Spamd-Result: default: False [-9.57 / 15.00]
 WHITELIST_SPF_DKIM(-3.00)
...
 R_DKIM_ALLOW(-1.10)
 FORGED_SENDER(0.30)
X-Spamd-Server: localhost
~~~
Would you post values of preferences: `mailnews.customDBHeaders`, `mailnews.customHeaders` and all begins with `extensions.rspamd-spamness.`?

**Comment by hufon** [comment]
mailnews.customDBHeaders : x-spamd-result
mailnews.customHeaders x-spamd-result

![shot](https://cloud.githubusercontent.com/assets/1788150/12728544/a000d37c-c922-11e5-8768-643bebd9d6dd.png)

**Comment by moisseev** [comment]
Looks fine except colons shouldn't be in mailnews.customDBHeaders. It is a  space delimited list. But it doesn't affect the add-on actually.
What add-on, Thunderbird, and OS versions are you using?

**Comment by moisseev** [comment]
The issue should be fixed in `master` with 75d7bfb.
Would you test it?

**Comment by hufon** [comment]
How could i test it? where could i download the xpi?

**Comment by moisseev** [comment]
Well, I've created pre-release [0.7.2](https://github.com/moisseev/rspamd-spamness/releases/tag/0.7.2)

**Comment by hufon** [comment]
Fixed for me!

Thanks!

**Comment by moisseev** [comment]
Fixed with 75d7bfb.
